detox/runners/jest/DetoxTraceAdapter.js

```diff
@@ -0,0 +1,66 @@
+const chalk = require('chalk').default;
+
+class DetoxTraceAdapter /* implements JasmineReporter */ {
```

Comment:
The naming is a bit confusing, as it is not an adapter, it is just a Jasmine reporter.

Reply:
I don't see any tight relation to Detox here, though the idea with the reporter looks justified and good. I suggest extracting it out of Detox.

See my comment here: https://github.com/wix/Detox/pull/1351#issuecomment-489398577